docs/documentation/full-text/phrase.mdx

```diff
@@ -39,3 +39,7 @@ SELECT description, rating, category
 FROM mock_items
 WHERE description @@@ '"plastic keyb"*';
 ```
+
+## Special Characters
+
+The special characters `+` , `^`, `` ` ``, `:`, `{`, `}`, `"`, `[`, `]`, `(`, `)`, `~`, `!`, `\\`, `\*`, and `SPACE` must be escaped by a`\` inside the query term.
```

Comment:
@philippemnoel Thanks for adding this clarification the doc! One question though - should `SPACE` be in this list for phrase? My understanding is that escaping whitespace **in a phrase** doesn't do anything:
`description:"word1 word2 word3"` is the same thing as `description:"word1\ word2\ word3"`

I tried playing around with adding the `~` operator to see if that makes `\ ` act any different than `  `, but doesn't seem so.
e.g. `description:"word1 word2 word4"~1` seems to behave the same as `description:"word1 word2\ word4"~1`  etc.

Reply:
Hmm... Good catch! You're probably right. Would you be willing to open up a PR to contribute this small doc fix? We'd love to give you credit for finding it :)